docs/src/introduction_to_catalyst/introduction_to_catalyst.md

```diff
@@ -306,53 +173,80 @@ k \frac{X^2}{2!} \frac{Y^3}{3!} \\
 giving the ODE model
 ```math
 \begin{align*}
-\frac{dX}{dt} &=  -2 k \frac{X^2}{2!} \frac{Y^3}{3!}, &
-\frac{dY}{dt} &=  -3 k \frac{X^2}{2!} \frac{Y^3}{3!}, &
+\frac{dX}{dt} &= -2 k \frac{X^2}{2!} \frac{Y^3}{3!}, &
+\frac{dY}{dt} &= -3 k \frac{X^2}{2!} \frac{Y^3}{3!}, &
 \frac{dZ}{dt} &= k \frac{X^2}{2!} \frac{Y^3}{3!}.
 \end{align*}
 ```
-This implicit rescaling of rate constants can be disabled through explicit
-conversion of a [`ReactionSystem`](@ref) to another system via
-[`Base.convert`](@ref) using the `combinatoric_ratelaws=false` keyword
-argument, i.e.
-```julia
-rn = @reaction_network ...
-convert(ODESystem, rn; combinatoric_ratelaws=false)
-```
+This implicit rescaling of rate constants can be disabled through [explicit providing the `combinatoric_ratelaws = false` argument](@ref ref) when an ODE is generated (e.g. through `ODEProblem`).
 
-For the previous example using this keyword argument would give the rate law
+For the previous example, using the `combinatoric_ratelaws = false` keyword argument would give the rate law
 ```math
 k X^2 Y^3
 ```
 and the ODE model
 ```math
 \begin{align*}
-\frac{dX}{dt} &=  -2 k X^2 Y^3, &
-\frac{dY}{dt} &=  -3 k X^2 Y^3, &
+\frac{dX}{dt} &= -2 k X^2 Y^3, &
+\frac{dY}{dt} &= -3 k X^2 Y^3, &
 \frac{dZ}{dt} &= k X^2 Y^3.
 \end{align*}
 ```
 
----
-## Notes
-1. For each of the preceding models we converted the `ReactionSystem` to, i.e.,
-   ODEs, jumps, or SDEs, we had two paths for conversion:
+## [Catalyst represents its models symbolically, and these can be generated programmatically](@id introduction_to_catalyst_sym_prog)
```

Comment:
This example is probably a bit long, but I thought it was really nice to be able to show of both automatic generation using programmatic modelling, _and_ how to utilise symbolic representations.